infra/build/functions/project_experiment.py

```diff
@@ -17,12 +17,17 @@
 
 import argparse
 import logging
+import os
 import sys
 
-import google.auth
-
 import build_lib
 import build_project
+import google.auth
+
+sys.path.append(
```

Comment:
This is a little hacky. We can avoid this by instead using the implementation here: https://github.com/google/oss-fuzz/blob/b5205aebc4caa161f16a98496dda820a02c83f1e/infra/build/functions/build_project.py#L211

Reply:
🙏